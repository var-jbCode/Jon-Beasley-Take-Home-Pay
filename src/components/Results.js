import React from 'react';

const Results = (props) => {
    return (
        <div className="results">
            <p><b>Your Take Home Pay is £{props.takeHome}</b></p>
            <p>Your Annual Tax due is £{props.annualTaxDue}</p>
            <p>Your Annual NI due is £{props.annualNIDue}</p>
        </div>

    );
}

export default Results;