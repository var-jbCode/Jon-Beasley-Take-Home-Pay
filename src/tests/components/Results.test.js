import ReactShallowRenderer from 'react-test-renderer/shallow'

import React from 'react'
import Results from '../../components/Results'

test('Should Render Results Correctly without Props', () => {
    const renderer = new ReactShallowRenderer()
    renderer.render(<Results />)
    expect(renderer.getRenderOutput()).toMatchSnapshot();
})
