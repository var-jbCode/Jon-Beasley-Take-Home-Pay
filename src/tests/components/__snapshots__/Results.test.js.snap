// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Results Correctly without Props 1`] = `
<div
  className="results"
>
  <p>
    <b>
      Your Take Home Pay is £
    </b>
  </p>
  <p>
    Your Annual Tax due is £
  </p>
  <p>
    Your Annual NI due is £
  </p>
</div>
`;
